symfony/plugins/orangehrmPimPlugin/Service/EmployeeDependentService.php

```diff
@@ -89,4 +92,32 @@ public function deleteEmployeeDependents(int $empNumber, array $entriesToDelete)
     {
         return $this->getEmployeeDependentDao()->deleteEmployeeDependents($empNumber, $entriesToDelete);
     }
+
+    /**
+     * @param EmployeeDependentSearchFilterParams $employeeDependentSearchParams
+     * @return array
```

Comment:
Can you update `@return array` to `@return EmpDependent[]`,
Update `getEmployeeDependents` here as well